src/Xamarin.Android.Build.Tasks/Utilities/Files.cs

```diff
@@ -271,9 +341,16 @@ public static ZipArchive ReadZipFile (string filename, bool strictConsistencyChe
 		}
 
 		public static string HashString (string s)
+		{
+			var bytes = Encoding.UTF8.GetBytes (s);
```

Comment:
Too bad BitConverter does not have span APIs yet :(

Reply:
`Encoding.GetBytes` has an overload where you can reuse a `byte[]`, but because this is a static method I would have to write some kind of `byte[]` pool that is thread safe... Didn't want to go that far yet.

It might be we should avoid the `string` overload when we can, which is still better than using a temp file.

Reply:
`ArrayPool<byte>`?

Reply:
YMMV, but you may need System.Buffers nuget (not sure if that's possible for this project)

Reply:
I wasn't aware of [`ArrayPool<T>`](https://docs.microsoft.com/en-us/dotnet/api/system.buffers.arraypool-1?view=netcore-2.2), but yeah if we opt into using `Span<T>` at some point we would have System.Buffers then.

I'll put a `//TODO:` comment for now.